codalab/lib/bundle_cli.py

```diff
@@ -1434,6 +1434,7 @@ def wrap(string):
         ),
     )
     def do_cat_command(self, args):
+        # TODO: Real?? fail_if_headless? What about wait.
         self._fail_if_headless('cat')  # Files might be too big
```

Comment:
This comment wasn't supposed to make it into the CL. =)

Yeah, I think it's worth it removing the check. I don't buy the "files might be too big" comment.